Automattic/wp-calypso
# Help Center: Re-design improvements

Related to https://github.com/Automattic/dotcom-forge/issues/9226
Fixes https://github.com/Automattic/dotcom-forge/issues/9226

## Proposed Changes

* Apply the new redesign changes CMvR9P4FontAbv2b0TNMzj-fi-282_49597 on Help Center.

## Why are these changes being made?


* Match the new design.

## Testing Instructions



* Use live link or pull and run this branch
* Make sure to add the feature flag to see the design changes, `?flags=help-center-experience`
* Start a new chat
* Compare against the designs CMvR9P4FontAbv2b0TNMzj-fi-282_49597
* Make sure to explore around, create a new chat, use Zendesk to reply to your own messages
* Check any possible regressions without the flag

## Pre-merge Checklist
